VipAccount.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "Customer.h"
+
+#define VIP_REWARD_EXCHANGE_CONDITION 100
+#define VIP_REWARD_POINTS 10
+
+class VipAccount : public Customer {
+private:
+  int rewardPoints;
+
```

Comment:
Suggestion:
- An attribute to track how many free rental he/she has in one session. In addition, rentItem() should be merge with freeRentItem() as we can check the VIP customer's attribute of free rental.

Reply:
Done